pyproject.toml

```diff
@@ -25,7 +25,7 @@ pytest-cov = { version = "2.10.1", optional = true }
 torch = { version = "1.13.0", optional = true }
 torchvision = {version = "*", optional = true }
 neptune = { version = ">=1.0.0", optional = true }
-
+numpy = { version = "<2.0", optional = true }
```

Comment:
why is it optional? Isn't that going to be ignored if we don't install via `pip install ...[numpy]`?